lib/query-utils.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const cleanQuery = (query) => {
+  const regex = /(\()[\sa-zA-Z0-9,\:\"\'\{\}\$\!]+(\))/g
```

Comment:
do we need all the specific checks in the middle here or could we just wildcard between two parens?

Reply:
Good point.  I'll try it out, but I think I had trouble with handling newlines with a wild card.  might work with /s/S